src/protocol-curr/xdr/Stellar-overlay.x

```diff
@@ -22,9 +22,37 @@ struct Error
     string msg<100>;
 };
 
-struct SendMore
+enum SendMoreType
```

Comment:
Heads up, this change isn't right - we need to still support old-style XDR for backward compatibility